src/app/legacy/components/ScrollablePromo/Promo/index.jsx

```diff
@@ -171,6 +172,10 @@ const Promo = ({ block, experimentVariant, onClick }) => {
 
   const WrapperPromoBox = isOperaMini ? OperaPromoBox : PromoBox;
 
+  const variation = useOptimizelyMvtVariation('top_bar_oj_experiment');
+
+  if (variation === 'off') return null;
```

Comment:
This block will need to sit above https://github.com/bbc/simorgh/blob/8fb0442577f1c0ea683e79778b861eece5e5e525/src/app/legacy/components/ScrollablePromo/Promo/index.jsx#L113

We'll need to call the variable the same name to ensure we hit the switch/case to properly determine the data to be used

Reply:
Theres also this block of logic that'll need to take into account the variation: https://github.com/bbc/simorgh/blob/2cff01d0a6cc6b3544553789cf3919185e909865/src/app/components/PageLayoutWrapper/index.tsx#L75-L105

@LilyL0u may be better placed to explain this logic and how we slot in Optimizely into here though

Reply:
This may be slightly tricky to use the `useOptimizelyMvtVariation` hook here, as it won't be wrapped in the `withOptimizelyProvider` that the ArticlePage is. `PageLayoutWrapper` sits at a higher level than the actual ArticlePage component.

Reply:
You could wrap the `PageLayoutWrapper` with `withOptimizelyProvider`, but that would mean _all_ page types get Optimizely. Not a massive problem, but it will add page weight to page types that aren't in experimentation.

Reply:
Actually if you can get the variation from Optimizely and set the `experimentVariant` value here: https://github.com/bbc/simorgh/blob/2cff01d0a6cc6b3544553789cf3919185e909865/src/app/components/PageLayoutWrapper/index.tsx#L89 then that is passed all the way down to the Scrollable promo along with the data.

That means you could move the `useOptimizelyMvtVariation` hook out of ScrollablePromo and up to PageLayoutWrapper.

Reply:
Works for me!
```
    const variantValue = useOptimizelyMvtVariation('top_bar_oj_experiment');
```

Means all this can be removed:
```
    const servicesForVariantA = ['pidgin', 'hindi', 'urdu', 'uzbek', 'hausa'];
    const servicesForVariantB = ['mundo', 'burmese', 'arabic'];

    let variantValue;

    if (servicesForVariantA.includes(service)) {
      variantValue = 'A';
    } else if (servicesForVariantB.includes(service)) {
      variantValue = 'B';
    } else {
      variantValue = 'none';
    }
```

Reply:
In fact, it could be simplified even further to:
```
    const experimentVariant = useOptimizelyMvtVariation(
      'top_bar_oj_experiment',
    );

    let dataForOJExperiment;
    if (experimentVariant === 'A') {
      dataForOJExperiment = topStories;
    } else if (experimentVariant === 'B' && mostReadItems) {
      dataForOJExperiment = mostReadItems;
    }

    propsForOJExperiment = {
      blocks: dataForOJExperiment || [],
      experimentVariant,
    };
```